pkg/cmd/tknpac/resolve/resolve.go

```diff
@@ -212,6 +213,8 @@ func resolveFilenames(ctx context.Context, cs *params.Run, filenames []string, p
 	}
 
 	for _, run := range prun {
+		run.APIVersion = tektonv1.SchemeGroupVersion.String()
```

Comment:
i see from the issue 
tkn pac resolve -f .tekton/pipelinerun.yaml -o output-file.yaml

what if .tekton/pipelinerun.yaml have `v1beta1` it will still add `v1` based on the code :thinking:

Reply:
we always generate a v1 since that's what we do on controller... i don't think we need to do it differently for the cli.,.